### PR TITLE
[News] Added The Ultimate List of Open Source Cloud-Native Tools

### LINKED ISSUE #3128: [News] The Ultimate List of Open Source Cloud-Native Tools
We're in the news! Using the contributing instructions for [adding news](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#adding-news), please add the following article to the [Layer5 Newsroom](https://layer5.io/news).

**Article:**
- Title: The Ultimate List of Open Source Cloud-Native Tools
- Link: https://containerjournal.com/features/the-ultimate-list-of-open-source-cloud-native-tools/

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by legendarykamal**
**Description**
Added The Ultimate List of Open Source Cloud-Native Tools

This PR fixes #3128 

**Notes for Reviewers**
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f277c416cd69f1453c78579f6e99fb0f81f7d403 at: https://631774807092690316fb21ab--layer5.netlify.app

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (debo19) [inline .gitpod.yml:1]: I think you've added the wrong file. Index file is missing.

File: src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: "The Ultimate List of Open Source Cloud-Native Tools"
+subtitle: ""
+date: 2022-08-29 00:00:08 +0000
```
Comment (debo19) [inline src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx:4]:
```suggestion
date: 2022-08-29 00:00:08 +0000
category: Coverage
```

Comment (debo19) [reply src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx:4]: This should fix the build fail

**Review by debo19 — CHANGES_REQUESTED** [review]

File: src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+<h3>Observability and Analysis</h3>
+Prometheus tops the list of observability and analysis tools. The platform can be used to power your monitoring and alerting systems with fine-grained metrics and excellent querying capabilities. Other CNCF tools for observability and analysis include Jaeger, Fluentd, Thanos, Cortex and OpenTelemetry.
+
+<h3>Security and Compliance</h3>
+Open Policy Agent (OPA) can be used to unify cloud-native policies across the cloud-native stack. OPA uses a common language to express authorization policies and provides a policy engine to make authorization decisions. Other CNCF projects that deliver security-as-code include The Update Framework (TUF), Falco, Notary, cert-manager and Curiefense.
+
+<h3>CI/CD</h3>
+Argo is a suite of packages that help direct jobs on Kubernetes to aid a continuous delivery pipeline. Using Argo, developers can create multi-step custom workflows and share these workflows across a cluster. Other CI/CD tools hosted by CNCF include Flux, Brigade, Keptn, OpenGitOps and OpenKruise.
+<h3>Service Mesh Tools</h3>
```
Comment (leecalcote) [inline src/collections/news/2022/2022-09-06-the-ultimate-list-of-opensource-cn-tools/index.mdx:36]: Add links to the /meshery and the /service-mesh-performance and the /projects/service-mesh-interface-conformance pages, please.

**Comment by l5io** [comment]
🚀 Preview for commit 1060a29ba624ba4a821ca2991db1b0a7770d2ac5 at: https://63195868530c54195b09819c--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 05d2bb98decc08f41a140714473fa38739389bdb at: https://6319ea3c9797eb008995ed1b--layer5.netlify.app

**Comment by leecalcote** [comment]
Please ensure that there is whitespace between sections and between section headings and their content.

Screen Shot 2022-09-08 at 8 28 40 AM


Don't forget the period after SMP.

Let's go ahead an include a copy of the full article in this post, please.

**Comment by legendarykamal** [comment]
ok @leecalcote  I thought Read the full article would lead but yes Lets include the full article :smiley:

**Comment by legendarykamal** [comment]
@leecalcote Included the full code coverage with minor new line changes ,  :+1:

**Comment by l5io** [comment]
🚀 Preview for commit 1d74b8f64074fc49ef74fdd8185bf79bedb9d790 at: https://631abe2389b56260b607af67--layer5.netlify.app

**Comment by leecalcote** [comment]
This is a bit odd - how is it that the single news page for a piece of news lists the same piece of news as the first related article at the bottom of the page? 

We might have a query issue.

**Comment by legendarykamal** [comment]
> This is a bit odd - how is it that the single news page for a piece of news lists the same piece of news as the first related article at the bottom of the page?
> 
> We might have a query issue.

yes @leecalcote I have personally tried to solve this and opened up a new issue for this . Related posts actually queries all the post that are latest by date hence it reflects below.

**Comment by leecalcote** [comment]
@legendarykamal, this is very much appreciated. Thank you for taking the initiative to enhance the behavior here.

**Pull request merged**